packages/shared/src/lib/styling.ts

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames';
+
+const fadedBackground =
```

Comment:
This seems like a lot of overhead I think we can simply get a new color match preferably?
I have the same issue now with feed filters and think we should rather ask design for a solid color?

Reply:
To iterate, perhaps it's better to add a new scope of colors "faded" and add all the new faded colors?
But suggestion is to ask design for all of them :)

Reply:
Yeah let's rather ask design.

They can give us 8 digit hex colors so we can adjust them in our guidelines rather.
I prefer having a className for it, compared to a function that needs to execute.

Reply:
Now that we have concluded the usage of opacity, I think we will have to move forward with the function to generate it until we upgrade our tailwind version.

We probably need to put some comments on it to make sure it gets updated. WDYT?

Reply:
As far as I was concerned the solution was actually to generate classes for it right?
I feel like we might be misaligned here from what feedback we received let's rather discuss it quickly today.

Reply:
Let's jump in Gather once you are available.